actions.md

```diff
@@ -362,36 +362,35 @@ These tools all link with libaktualizr, although they do not necessary use the A
 
 `aktualizr-cert-provider` assists with generating credentials and uploading them to a device for implicit provisioning.
 
-- [ ] Use file paths from config if provided
-- [ ] Use autoprovisioning credentials if fleet CA and private key are not provided
+- [x] Use file paths from config if provided (cert_provider_test.cc)
+- [x] Use autoprovisioning credentials if fleet CA and private key are not provided (cert_provider_test.cc)
```

Comment:
Some of these are now in cert_provider_autoprov_test.cc now.

Reply:
done